### DISCUSSION TITLE
通过 qiankun 这个库可以安全地执行任意不可信的代码吗？

### DISCUSSION THREAD

**Discussion by ghost** [umijs/qiankun#2698] [General]
这两天在搜索如何安全地运行不可信的 JavaScript 代码的时候了解到了 qiankun。我想实现一个基本就是 [WebOS](https://edutechwiki.unige.ch/en/Web_operating_system) 的东西。这是个 WebOS  的例子： https://demo.kodcloud.com/ 。

**Comment by kuitos** [comment]
不行，iframe都不行。微前端主要是面向二方开发场景的，不是面向三方的
